google/cloud/spanner/samples/samples.cc

```diff
@@ -391,6 +391,27 @@ void DmlStandardDelete(google::cloud::spanner::Client client) {
 }
 //! [END spanner_dml_standard_delete]
 
+//! [execute-sql-partitioned] [START spanner_dml_partitioned_delete]
+void DmlPartitionedDelete(google::cloud::spanner::Client client) {
+  namespace spanner = google::cloud::spanner;
+  auto result = client.ExecutePartitionedDml(
+      spanner::SqlStatement("DELETE FROM Singers WHERE SingerId > 10"));
+  if (!result) throw std::runtime_error(result.status().message());
+  std::cout << "Delete was successful [spanner_dml_partitioned_delete]\n";
+}
+//! [execute-sql-partitioned] [END spanner_dml_partitioned_delete]
+
+//! [START spanner_dml_partitioned_update]
+void DmlPartitionedUpdate(google::cloud::spanner::Client client) {
+  namespace spanner = google::cloud::spanner;
+  auto result = client.ExecutePartitionedDml(
+      spanner::SqlStatement("UPDATE Albums SET MarketingBudget = 100000"
```

Comment:
How does this sample know `MarketingBudget` exists?

Reply:
The same way all the examples know that the tables exist, or any other column. They need to be executed in a particular order.

Reply:
But `samples.cc` also has `RunOneCommand` support.

Reply:
Yes. And it was true for other existing examples that use `MarketingBudget`, and it is also true in the samples for other languages (and libraries).  I get the point, it means the samples can fail if you don't setup your environment "just so", but (I think) this is not the PR to fix that (assuming we want to fix it).

Reply:
Fair enough, but I wasn't suggesting this was the PR to fix it ... just the PR not to introduce another occurrence.